benchmarks/tools/FileOnQ.Imaging.Heif.Benchmarks.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFrameworks>net48;net5.0;net6.0</TargetFrameworks>
```

Revert benchmark changes
```suggestion
    <TargetFramework>net5.0</TargetFramework>
```